### DISCUSSION TITLE
ESP-32 E-Paper support UC8151D, SSD1680

### DISCUSSION THREAD

**Discussion by abeginnDev** [olikraus/u8g2#1447] [General]
Hello, 
i have 2 E-paper Displays from Good-Display. The GDEW029M06 with UC8151D and the GDEM029T94 with SSD1680. I got used to the Library and want to use it for them to. Is there any Support for them or planned in the future? Also i want to say thank you for the good Documentation.

**Comment by cblack3** [comment]
You should check out the [U8g2_for_Adafruit_GFX](https://github.com/olikraus/U8g2_for_Adafruit_GFX), both of those drivers are supported by Adafruit GFX.

**Reply by olikraus** [reply]
Actually I stopped working on e-paper devices with u8g2. There are better libraries instead like the mentioned Adafruit GFX. You could still use u8g2 fonts with the https://github.com/olikraus/U8g2_for_Adafruit_GFX library.
